base/ltproperties.dtx

```diff
@@ -746,9 +746,13 @@
 %    \end{macrocode}
 % \end{macro}
 %
-% \begin{macro}{\IfPropertyExistTF}
+% \begin{macro}{\IfPropertyExistsTF,\IfPropertyExistsT,\IfPropertyExistsF}
+% \changes{v1.0e}{2024-04-17}{Reanmed \cs{IfPropertyExistTF} to
+%                             \cs{IfPropertyExistsTF} (gh/1262)} 
 %    \begin{macrocode}
-\cs_new_eq:NN \IfPropertyExistTF \property_if_exist:eTF  
+\cs_new_eq:NN \IfPropertyExistsTF \property_if_exist:eTF  
+\cs_new:Npn   \IfPropertyExistsT #1#2 {\property_if_exist:eTF {#1}{#2}{} }  
+\cs_new:Npn   \IfPropertyExistsF #1   {\property_if_exist:eTF {#1}{} }  
```

Comment:
Do we not have `expl3` level `\property_if_exist:eT`, etc?

Reply:
Ah yes, only `TF` is generated.
https://github.com/latex3/latex2e/blob/a46f5273447b9a26b518272ec5a510c2c6136ed5/base/ltproperties.dtx#L745

Reply:
Probably should change that then - I think it's odd to have the CamelCase ones but not the `expl3` ones

Reply:
OK, I can live with that :)